volatility3/framework/layers/linear.py

```diff
@@ -69,3 +74,14 @@ def _scan_iterator(self,
                        sections: Iterable[Tuple[int, int]],
                        linear: bool = True) -> Iterable[interfaces.layers.IteratorValue]:
         return super()._scan_iterator(scanner, sections, linear)
+
+    @classmethod
+    def is_address_canonical(cls, address: int) -> bool:
```

Comment:
Why is this only on linear layers?  It feels either like it iws either something you might want to be able to call on all layers, or just on Intel?

Reply:
In the future the ARM, PPC and others will inherit from this class just like the `Intel` layer.
It is not Intel specific and will work for other architectures.